{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "           AssertionError: Unexpected message: User profile updated successfully.\n           assert {'accessFailedCount': 0, 'concurrencyStamp': '2a980af1-4984-44b7-9c3b-62ff3351cd6a', 'countryCodeID': None, 'createdDate': '2025-01-23T02:12:09.243', ...} == 'User profile updated successfully.'\n            +  where {'accessFailedCount': 0, 'concurrencyStamp': '2a980af1-4984-44b7-9c3b-62ff3351cd6a', 'countryCodeID': None, 'createdDate': '2025-01-23T02:12:09.243', ...} = <built-in method get of dict object at 0x0000015BE9B9A800>('responseData')\n            +    where <built-in method get of dict object at 0x0000015BE9B9A800> = {'isSuccess': True, 'message': 'User profile updated successfully.', 'responseData': {'accessFailedCount': 0, 'concurrencyStamp': '2a980af1-4984-44b7-9c3b-62ff3351cd6a', 'countryCodeID': None, 'createdDate': '2025-01-23T02:12:09.243', ...}}.get\n", "test_name": "test_update_user_profile_and_profile_password", "rerun": "0"}}, "suite_name": "main/test_auth.py"}}}, "date": "January 23, 2025", "start_time": 1737630643.9760926, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}